src/redux/reducers/ConceptReducers.js

```diff
@@ -143,11 +143,11 @@ export default (state = initialState, action) => {
         ...state,
         description: filterNames(action.payload, state.description),
       };
-    case ADD_NEW_ANSWER_MAPPING:
-      return {
-        ...state,
-        answer: [...state.answer, action.payload],
-      };
+    // case ADD_NEW_ANSWER_MAPPING:
```

Comment:
@AGMETEOR , kindly remove the commented code.

Reply:
@Duncanian I have removed it.